wallet/src/de/schildbach/wallet/service/platform/PlatformSyncService.kt

```diff
@@ -1142,10 +1142,11 @@ class PlatformSynchronizationService @Inject constructor(
             currentRequestList.forEach { request ->
                 val voteContender = platformRepo.getVoteContenders(request.normalizedLabel)
                 if (voteContender.winner.isPresent) {
+                    // remove request
                     usernameRequestDao.remove(request.requestId)
+                    // remove related votes
+                    usernameVoteDao.remove(request.normalizedLabel)
                 }
-                // remove related votes
-                usernameVoteDao.remove(request.normalizedLabel)
```

Comment:
Before our records of username votes were getting removed even though voting wasn't finished.